config/pre-cache/overrides.yaml

```diff
@@ -13,4 +13,6 @@ data:
   #   performance-addon-operator: 4.9
   #   ptp-operator: stable
   #   sriov-network-operator: stable
-
+  # excludePrecachePatterns: |
+  #   aws
```

Comment:
@vitus133 @imiller0 Any other common patterns should be included in the example?